File: src/main/java/com/school/app/CodeLoader.java
package com.school.app;

import com.school.domain.code.rest.Class;

public class CodeLoader {
    public Class load(String taskId) {
        String code = "package javaschool.task.section01.task01;\n" +
                "\n" +
                "public class Main {\n" +
                "    public static void main(String[] args) {\n" +
                "\n" +
                "    }\n" +
                "}";

        return new Class(code, "Main");
    }
}

File: src/main/resources/static/scripts/code-loader.js
$(document).ready(function() {
    $.ajax({
        url:'/task/code',
        type:'get',
        data: {
            taskId: 'section01.task01'
        }
    }).done(function(response) {
        handleResponse(JSON.parse(response));
    }).fail(function() {
        alert('Oops, something is wrong...')
    });


    function handleResponse(response) {
        ace.edit("code-editor").setValue(response.code, 1);
    }
});